test/testUtils.js

```diff
@@ -60,6 +60,12 @@ export function matchRequest(
       url,
     } = moxios.requests.mostRecent().config
 
-    return { data, headers, method, params, url }
+    return {
+      data: data !== undefined ? JSON.parse(data) : undefined,
```

Comment:
This _will_ break existing tests but the tests will also be more readable once this version is deployed.

Look at the snapshot from this diff: https://github.com/skedify/sdk-js/pull/23/commits/5f474408fa03a77b54bb0a6f997719d038d0602a#diff-2557f2dca7cb9cc95f83d70070994f6b